kolibri/plugins/device/assets/src/views/ManageContentPage/SelectNetworkAddressModal/index.vue

```diff
@@ -5,6 +5,7 @@
       v-if="stage === Stages.SELECT_ADDRESS"
       @cancel="resetContentWizardState"
       @click_add_address="goToAddAddress"
+      @click_search_address="goToSearchAddress"
```

Comment:
conventionally we use `camelCase` for events. I see that this matches the others in the component so this is fine, but would be nice to refactor later for consistency

Reply:
Good catch.  I didn't bother to go that in-depth.  That code is copied from another file and looking at it now, is going to be deleted anyway.